cmd/finch/nerdctl_windows.go

```diff
@@ -107,21 +106,250 @@ func handleVolume(systemDeps NerdctlCommandSystemDeps, args []string, index int)
 	return nil
 }
 
+// translates source path of the bind mount to wslpath for --mount option
+func handleBindMounts(systemDeps NerdctlCommandSystemDeps, args []string, index int) error {
+	var prefix = args[index]
+	var v = ""
+	var found = false
+	var before = ""
+	if strings.Contains(args[index], "=") {
+		before, v, found = strings.Cut(prefix, "=")
+	} else {
+		if (index + 1) < len(args) {
+			v = args[index+1]
+		} else {
+			return fmt.Errorf("invalid positional parameter for %s", prefix)
+		}
+	}
+
+	// https://docs.docker.com/storage/bind-mounts/#choose-the--v-or---mount-flag  order does not matter, so convert to a map
+	entries := strings.Split(v, ",")
+	m := make(map[string]string)
```

Comment:
Any thoughts on using a struct for this instead of a map? nerdctl / buildkit / docker-cli code might have something we can just import? It just might make it clearer if we can enumerate all of the options on a mount and check if they're empty, instead of just checking if a map has a key